### ISSUE TITLE
Why is DoH disabled by default?

### ISSUE THREAD

**Issue by flexagoon** [arkenfox/user.js#1382]
I don't understand why Arkenfox does not enable DNS over HTTPS by default. Is there something wrong with the current implementation of it in Firefox? I thought that DoH is always a good thing, unless you need to inspect network traffic. Is this not the case?

**Comment by Thorin-Oakenpants** [comment]
Because it is something that the user needs to decide, not AF unilaterally imposing some 3rd party on everyone

**Comment by Thorin-Oakenpants** [comment]
I got two notifications (email) - the first for here and a second for a new issue: they were the same post. I deleted the other issue (I consider it to be spam and a duplicate) and then I see the original post is now not here.

That original post was a about getting a prompt for DoH and clicking the `x` close button (instead of reading)

For the same reason we don't set DoH, we also do not block any rollout - because there is a prompt. If we did, that would rob users of the decision making process. That prompt is default opted-in - and I have no issue with that - if it was opt-in people complain, if it was opt-out people complain. DoH is not a bad thing. And I will note that it has only been rolled out to countries with generally shitty ISPs, IMO, so that's a good thing - default opted-in in "shitty" countries for the  great dirty unwashed masses. Also, people need to learn to read

There is no dark pattern, see the pic at https://support.mozilla.org/en-US/kb/firefox-dns-over-https - it clearly says you were opted in and the default button is `Okay` - to opt-out you clearly need to click `Disable`

tl;dr:  it is something that the user needs to decide - AF will not enable DoH and will not block DoH
